src/components/grid-aware/HomePageLargeParagraph/HomePageParagraph.module.css

```diff
@@ -0,0 +1,35 @@
+.bleedWrapper {
+  justify-items: center;
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+  padding-bottom: 120px;
+  padding-top: 120px;
+}
+
+.bleedMainContent {
+  grid-column: 2;
+  grid-row: 1;
+  /* Override grid item default of auto so that item contents cannot stretch the grid itself */
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    grid-template-columns: 40px 1fr 40px;
+  }
+}
+
+.title {
+  font: var(--font-headline);
+  margin-bottom: 40px;
+}
+
+
+.description {
+  font: var(--font-title);
+  text-align: center;
+  width: 1020px;
```

Comment:
Please remove this line, since this component should be receiving its width automatically from its parent. This is an example of a space-filling block component, per my taxonomy here: https://github.com/ShelterTechSF/sheltertech.org/blob/559e1cf695ed06c1d3d6f38e14b23d4cd6e5f102/docs/component-design-system.md#space-filling-block-components

An example of a problem that this causes is that in a mobile design, it causes the width to stretch beyond the width of the screen:

<img width="698" alt="Screen Shot 2020-10-25 at 6 27 14 PM" src="https://user-images.githubusercontent.com/1002748/97125043-c56a3980-16ef-11eb-8011-5fd9cc58cdcc.png">

Reply:
Sorry, I didn't take the mobile format into consideration. If I took the width line out, what is another way to format it so it looks like how it does in Figma?

Reply:
Ah, I see. Maybe then you should set the grid columns for this component such that it spans columns 2 through 11, inclusive?